test/test-helper.ts

```diff
@@ -43,7 +43,7 @@ export class TestDriver {
   }
 
   synth = () => {
-    execSync(`cdktf synth`, { stdio: "inherit", env: this.env });
+    execSync(`cdktf synth`, { stdio: "pipe", env: this.env });
```

Comment:
does this affect other existing tests in case of an error?

Reply:
Good catch. Just had a look, it swalled the syntax error I added for testing..
I'll see what I can do to make sure the child process keeps printing to stdout / err.